s3/replication/common/src/s3replicationcommon/s3_head_object.py

```diff
@@ -90,12 +436,12 @@ async def get(self):
                     headers=headers) as resp:
 
                 if resp.status == 200:
-                    self._response_headers = resp.headers
+                    self._response_headers = dict(resp.headers)
                     self._logger.info(
```

Comment:
manually format and reduce the number of lines used by this statement and check for other places too.

Reply:
done